app/views/users/show.html.erb

```diff
@@ -1,7 +1,13 @@
-<h1><%= @user.name %></h1>
+
+<div class="show-user">
+<div class="container"><div class="cadre"></div><div class="user"><h2><%= @user.name  %></h2> <p><span></span><span>Number of posts : <span><%= @user.posts_counter %></span></span></p></div></div>
+
+<p class='bio'><%= @user.bio %></p>
 <br>
-<p><%= @user.bio %></p>
+<% @post_most_recent.each_with_index do |posts, idx|%>
+    <div class='post'><span class='title'><%= link_to 'Post #', user_posts_path(@user, posts) %><%= idx + 1 %></span>
+    <br><%= posts.text %></div>
+<% end %>
 <br>
-<%= @user.posts_counter %>
-<br>
-<%= link_to 'Go to Posts', users_path %>
```

Comment:
- [ ] As the previous reviewer mentioned, kindly ensure that on your `User Show` page, there is a button that takes users to view all posts for a user. :+1: 
- [ ] From the given wireframe, you can see that the button to `See all posts` was present beneath the 3 recent posts so kindly implement that and ensure that it opens to show the `posts index page` as seen in the given wireframe :+1:
- The Given wireframe for the `Users Show Page`
![image](https://github.com/francksefu/WebApp/assets/102699958/bb2d19ab-2fb8-4e90-aeea-07d7d4425e49)
 - Your Design
![image](https://github.com/francksefu/WebApp/assets/102699958/7c07d2b4-17a8-430f-a0dd-cba8b1f59eaf)